doc/source/whatsnew/v1.3.3.rst

```diff
@@ -17,7 +17,7 @@ Fixed regressions
 - Fixed regression in :class:`DataFrame` constructor failing to broadcast for defined :class:`Index` and len one list of :class:`Timestamp` (:issue:`42810`)
 - Performance regression in :meth:`core.window.ewm.ExponentialMovingWindow.mean` (:issue:`42333`)
 - Fixed regression in :meth:`.GroupBy.agg` incorrectly raising in some cases (:issue:`42390`)
--
+- Fixed regression in :meth:`merge` where columns with ``ExtensionDtype`` was cast to ``object`` in ``left`` and ``outer`` merge (:issue:`40073`)
```

Comment:
Maybe specify that the cast only occurs when those columns were being merged on?

Reply:
IIUC what happened here was that ``left`` and ``outer`` were always broken, so fixing those could be moved down into the bug fix section of the whatsnew.

Then the regression portion could just focus on ``right``

Reply:
my bad, this bug was pertaining to `right`(regression) and `outer` (had mentioned in the PR subject), also discussed in the bug comments. 
I will keep `right` in fixed regression, and move `outer` down to bug fix.